controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/rotation/Rotation.java

```diff
@@ -1,41 +0,0 @@
-// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
-package com.yahoo.vespa.hosted.controller.api.rotation;
-
-import com.yahoo.vespa.hosted.controller.api.identifiers.RotationId;
-
-import java.util.Objects;
-
-/**
- * Represents a global routing rotation.
- *
- * @author Oyvind Gronnesby
- */
-// TODO: Used in serialization (ConfigServerClient). This should be removed and config server client should use a
-// Set<String> instead, like it does for CNAMEs.
-public class Rotation {
```

Comment:
Please double-check that this is unused in internal repo before removing.

Reply:
I removed the usages in hosted. Are there other repos I need to worry about?

Reply:
Don't think so.